herbert_core/classes/MPIFramework/@parallel_config/parallel_config.m

```diff
@@ -215,22 +215,19 @@
         %-----------------------------------------------------------------
         % overloaded getters
         function wrkr = get.worker(obj)
-            wrkr= get_or_restore_field(obj,'worker');
+            wrkr = get_or_restore_field(obj, 'worker');
         end
         function wrkr = get.is_compiled(obj)
-            % incomplete! Should be derived from worker
-            wrkr= obj.is_compiled_;
+            wrkr = get_or_restore_field(obj, 'is_compiled');
```

Comment:
Yeah, this is just the `get` function - there is no `set` for `is_compiled` it can only be set by setting `worker`. 

If you set `worker` to something which is found by _Matlab_'s `which` (implying it is a Matlab function) then `is_compiled` is `false`. If you set `worker` to something which is not found by _Matlab_'s `which` but is a fully qualified path to an existing file, _or_ is something found by the _system_ `which` (`where` on Windows) then `is_compiled` is `true`.